content/id/docs/concepts/scheduling/scheduler-perf-tuning.md

```diff
@@ -0,0 +1,161 @@
+
+### _Threshold default_
+
+Jika kamu tidak mengatur sebuah _threshold_, maka Kubernetes akan melakukan
+perhitungan menggunakan pendekatan linier, yaitu 50% untuk klaster dengan 100 Node,
+serta 10% untuk klaster dengan 5000 Node.
+
+Artinya, kube-scheduler selalu menilai paling tidak 5% dari klaster kamu, terlepas dari
+seberapa besar klasternya, kecuali kamu secara eksplisit mengatur `percentageOfNodesToScore`
+untuk lebih kecil dari 5.
+
+Jika kamu ingin penjadwal untuk memasukkan seluruh Node di dalam klaster ke dalam penilaian, 
+maka atur `percentageOfNodesToScore` menjadi 100.
+
+## Contoh
+
+Contoh konfigurasi di bawah ini mengatur `percentageOfNodesToScore` menjadi 50%.
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1alpha1
+kind: KubeSchedulerConfiguration
+algorithmSource:
+  provider: DefaultProvider
+
+...
+
+percentageOfNodesToScore: 50
+```
+
+
+##  Menyetel percentageOfNodesToScore
+
+`percentageOfNodesToScore` merupakan angka 1 sampai 100 dengan
+_default_ yang dihitung berdasarkan ukuran klaster. Kamu juga bisa meng-_hardcode_
+menjadi angka minimal 50 Node.
+
+{{< note >}}Pada klaster dengan kurang dari 50 Node-Node layak, penjadwal masih
+terus memeriksa seluruh Node karena Node-Node layak belum mencukupi supaya
+penjadwal dapat menghentikan proses pencarian lebih awal.
+
+Pada klaster kecil, jika kamu mengatur `percentageOfNodesToScore` dengan angka kecil,
+pengaturan ini hampir atau sama sekali tidak berpengaruh, karena alasan yang sama.
+
+Jika klaster kamu punya ratusan Node, gunakan angka _default_ untuk opsi konfigurasi ini.
+Mengubah angka kemungkinan besar tidak akan mengubah performa penjadwal secara berarti.
+{{< /note >}}
+
+Sebuah catatan penting yang perlu dipertimbangkan saat mengatur angka ini adalah
+ketika klaster dengan jumlah Node sedikit diperiksa untuk kelayakan, beberapa Node
+tidak dikirim untuk penilaian sebuah Pod. Hasilnya, sebuah Node yang mungkin memiliki
+nilai lebih tinggi untuk menjalankan Pod bisa saja tidak diteruskan ke fase penilaian.
+Hal ini berdampak pada penempatan Pod yang kurang tepat.
+
+Kamu harus menghindari pengaturan `percentageOfNodesToScore` menjadi sangat rendah,
+sehingga kube-scheduler tidak seringkali memberikan keputusan penempatan Pod yang tidak bagus.
+Hindari pengaturan persentase di bawah 10%, kecuali _throughput_ penjadwal sangat penting
+untuk aplikasi kamu dan penilaian Node tidak begitu penting. Dalam kata lain, kamu
+memilih untuk menjalankan Pod pada Node manapun selama itu layak.
+
+## Bagaimana Penjadwal Mengecek Node
+
+Bagian ini ditujukan untuk kamu yang ingin mengerti bagaimana fitur ini bekerja secara internal.
+
+Untuk memberikan semua Node di dalam klaster sebuah kesempatan yang adil untuk
+dipertimbangkan dalam menjalankan Pod-Pod, penjadwal mengecek Node-Node satu persatu
+secara _round robin_. Kamu dapat membayangkan Node-Node ada di dalam sebuah _array_.
+Penjadwal mulai dari indeks _array_ pertama dan mengecek kelayakan dari Node sampai
```

Comment:
```suggestion
Penjadwal mulai dari indeks array pertama dan mengecek kelayakan dari Node sampai
```

Reply:
Fixed in https://github.com/kubernetes/website/pull/20260/commits/3cc163e65705127d1fcf16feba108179f9bdacc2